components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -51,9 +46,19 @@
           <div class="filter-label">Unknown</div>
           <div class="filter-total">{{ serviceGroupsHealthSummary.unknown }}</div>
         </chef-option>
+        <chef-option class="filter connected" value='connected' (click)="updateHealthFilter('connected')" (keyup.enter)="updateHealthFilter('connected')">
+          <chef-icon class="filter-icon">link</chef-icon>
+          <div class="filter-label">Connected</div>
+          <div class="filter-total">{{ serviceGroupsHealthSummary.total - serviceGroupsHealthSummary.disconnected }}</div>
+        </chef-option>
+        <chef-option class="filter disconnected" value='disconnected' (click)="updateHealthFilter('disconnected')" (keyup.enter)="updateHealthFilter('disconnected')">
+          <chef-icon class="filter-icon"></chef-icon>
+          <div class="filter-label">Disconnected</div>
+          <div class="filter-total">{{ serviceGroupsHealthSummary.disconnected }}</div>
+        </chef-option>
       </chef-status-filter-group>
     </div>
-    <div class="bulk-action-bar">
+    <div class="bulk-action-bar" [class.hidden]="(services$ | async)?.length === 0" [attr.aria-hidden]="(services$ | async)?.length === 0">
```

Comment:
I noticed this checkbox/button was still visible even when there are no services to be selected.  This just hides the checkbox/button for UX purposes when theres none to be selected anyways.

Reply:
💯